Mathlib/Analysis/Asymptotics/Asymptotics.lean

```diff
@@ -2137,6 +2140,11 @@ theorem IsLittleO.right_isBigO_add {f₁ f₂ : α → E'} (h : f₁ =o[l] f₂)
   ((h.def' one_half_pos).right_le_add_of_lt_one one_half_lt_one).isBigO
 #align asymptotics.is_o.right_is_O_add Asymptotics.IsLittleO.right_isBigO_add
 
+theorem IsLittleO.left_isBigO_add {f₁ f₂ : α → E'} (h : f₂ =o[l] f₁) :
+    f₁ =O[l] (f₁ + f₂) := by
+  rewrite [add_comm]
+  exact h.right_isBigO_add
```

Comment:
Here `right` in the original lemma refers to the fact that `f₁` that appeared on the right in the assumption now appears before `=O[l]`. I would call your lemma `right_isBigO_add'`.